docs/op-guide/recommendation.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Software and Hardware Requirements
+summary: Learn the software and hardware requirements for deploying and running TiKV.
+category: operations
+---
+
+# Software and Hardware Requirements
+
+As an open source distributed Key-Value database with high performance, TiKV can be deployed in the Intel architecture server and major virtualization environments and runs well. TiKV supports most of the major hardware networks and Linux operating systems.
```

Comment:
We also need to mention that TiKV must work together with PD and explain the relationship between them.